src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/FinremCaseData.java

```diff
@@ -326,6 +328,13 @@ public class FinremCaseData {
     private IntervenerFour intervenerFour;
     @JsonUnwrapped
     @Getter(AccessLevel.NONE)
+    private FormAScannedDocWrapper formAScannedDocWrapper;
+    @JsonUnwrapped
+    @Getter(AccessLevel.NONE)
+    private ConsentOrderScannedDocWrapper consentOrderScannedDocWrapper;
+    private List<ScannedD81Collection> scannedD81WithInfos;
```

Comment:
Could we call this scannedD81Collection please

Reply:
renamed